File: MyBlogDAL/Entities/Blog.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MyBlogDAL.Entities
{
    public class Blog : BaseEntity
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public string CreatorId { get; set; }
        public virtual User Creator { get; set; }
        public virtual ICollection<Article> Articles { get; set; }
    }
}

File: MyBlogAngular/src/app/models/blog.ts
export class Blog {
  id: number;
  name: string;
  description: string;
  creatorId: string;
  creatorName: string;

  constructor(id: number, 
    name: string, 
    description: string, 
    creatorId: string, 
    creatorName: string) {
      
    this.id = id;
    this.name = name;
    this.description = description;
    this.creatorId = creatorId;
    this.creatorName = creatorName;
  }
}